hotel-management/src/components/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode } from 'react';
+import './index.css';
+
+type ErrorMessageProps = {
+  color: 'primary' | 'error';
+  children: ReactNode;
```

Comment:
Why need to pass whole ReactNode,  I thought error message we need is just a message